src/scripts/EasternKingdoms/BlackrockMountain/MoltenCore/boss_ragnaros.cpp

```diff
@@ -83,6 +84,7 @@ class boss_ragnaros : public CreatureScript
                 _hasSubmergedOnce = false;
                 _isBanished = false;
                 me->SetUInt32Value(UNIT_NPC_EMOTESTATE, 0);
+				me->SetOrientation(0.8f);
```

Comment:
please fix indentation

Reply:
The hammer is still not facing the right way, but it appears to have moved in the right direction from where it used to fall.

Reply:
There is now a LARGE glowing orb of fire and some blood spatter falling down above the hammer which should not be there.